ldk/javascript/src/types/oliveHelps/whisper.d.ts

```diff
@@ -187,6 +199,19 @@ declare namespace WhisperService {
     componentStyle?: StyleOptions;
   };
 
+  type Grid = Component<'grid'> & {
+    alignContent?: AlignContent;
+    alignItems: AlignItems;
+    children: Array<ChildComponents>;
+    container: boolean;
+    direction?: GridDirection;
+    justifyContent?: JustifyContent;
+    item: boolean;
+    spacing: number;
+    wrap: Wrap;
+    xs?: string;
+  };
```

Comment:
I'm not sure if `container` and `item` should be required, in MUI they're mutually exclusive and technically optional. If this is required by the UI we should probably request that be changed.

I don't think `alignItems`, `spacing`, and `wrap` shouldn't be required either.

Reply:
@JosefBud-olive  `container` and `item` are required from UI. But there's logic on UI to deal with this relationship, so I think it's fine that they are required here? 🤔

Reply:
There's also logic to deal with it if they don't provide either one. In my opinion it would be better a developer experience to write syntax like

```jsx
<oh-grid container>
  <oh-grid item>
  <oh-grid item>
</oh-grid>
```

Rather than

```jsx
<oh-grid container item={false}>
  <oh-grid container={false} item>
  </oh-grid>
</oh-grid>
```

Since this component is supposed to mostly match the MUI Grid component, it would also be more in line with that.